runtime_scan/pkg/provider/aws/instance.go

```diff
@@ -32,21 +32,52 @@ type InstanceImpl struct {
 	ec2Client        *ec2.Client
 	id               string
 	region           string
+	vpcID            string
+	securityGroups   []string
 	availabilityZone string
+	image            string
+	ec2Type          string
+	platform         string
+	tags             []types.Tag
+	launchTime       time.Time
 }
 
 func (i *InstanceImpl) GetID() string {
 	return i.id
 }
 
 func (i *InstanceImpl) GetLocation() string {
-	return i.region
+	ret := i.region + "/" + i.vpcID
+	for _, group := range i.securityGroups { // TODO how do we want to represent multiple sg?
+		ret = ret + "/" + group
+	}
```

Comment:
maybe we should not put sg as part of the location, after all if instance can have multiple sg it doesn't say it exists in multiple locations.

Reply:
it's not correct to concat the security groups as it's not one within the other. 
I think that it's better to move the Security groups out of the location into a new property of the instance (list of SGs). WDYT?

Reply:
sgtm